tinyblog/articles/2016-03-28-5k-run.md

```diff
@@ -0,0 +1,19 @@
+# DjangoCon Europe 5k run
+
+> Published on 28th Mar 2016 by Tomáš Ehrlich
+
+The confrence is almost here and we're all super excited about it! 
+5 days of programming, networking and mostly fun.
+
+Let's gather together in the morning, before the conference to
+get a shot of pure oxygen! There is a beatiful Margaret Island 
+(Margitsziget) with even more beautiful park only 3 km or 15 mins
+from the venue. 
+
+We can meet on **31th March, at 7:00** near the **Margitsziget** 
+bus station (on the island).
+
+There is a [5 km long route](https://www.strava.com/segments/1056549) around the island,
+so we can all enjoy it at very peaceful pace.
```

Comment:
>so we can all enjoy it…

This makes it sound like there's a causal relationship between the route being there and the ability to run slowly :)

Not 100% sure what we tried to mean here, but how about 'There is a running track around the island. It's only 5 km long, so we can all enjoy it at a very peaceful pace and still get to the conference on time.' instead?